src/Logging.XUnit/XUnitLoggerOptions.cs

```diff
@@ -23,6 +25,12 @@ public XUnitLoggerOptions()
         /// </summary>
         public Func<string?, LogLevel, bool> Filter { get; set; } = (c, l) => true; // By default log everything
 
+        /// <summary>
+        /// Gets or sets the message sink message factory to use when writing to a <see cref="IMessageSink"/>.
+        /// By default, creates a <see cref="DiagnosticMessage"/>.
```

Comment:
As it returns an interface, let's not put the implementation detail in the comment as otherwise someone might decide that's a "contract" and complain in the future if it were changed for some reason.

```suggestion
```